integrations/va-auth0/src/index.tsx

```diff
@@ -89,83 +107,104 @@ const configBlock = createComponent<Auth0Props, Auth0State, Auth0Action, Auth0Ru
                         />
-                    }
-                />
-            </block>
+
+                        <input
+                            label="Client ID"
+                            hint={
+                                <text>
+                                    The unique identifier of your Auth0 application.
+                                    <link
+                                        target={{
+                                            url: 'https://auth0.com/docs/get-started/applications/application-settings',
+                                        }}
+                                    >
+                                        {' '}
+                                        More Details
+                                    </link>
+                                </text>
+                            }
+                            element={<textinput state="client_id" placeholder="Client ID" />}
+                        />
+
+                        <input
+                            label="Client Secret"
+                            hint={
+                                <text>
+                                    The secret used for signing and validating tokens.
+                                    <link
+                                        target={{
+                                            url: 'https://auth0.com/docs/get-started/applications/application-settings',
+                                        }}
+                                    >
+                                        {' '}
+                                        More Details
+                                    </link>
+                                </text>
+                            }
+                            element={
+                                <textinput state="client_secret" placeholder="Client Secret" />
+                            }
+                        />
+
+                        <input
+                            label="Add Callback URL"
+                            hint={
+                                <text>
+                                    Add this URL to the{' '}
+                                    <text style="bold">Allowed Callback URLs</text> section in your
+                                    application's settings in Auth0.
+                                </text>
+                            }
+                            element={<codeblock content={VACallbackURL} />}
+                        />
+                    </vstack>
+                    <divider size="medium" />
+
+                    <markdown content="### Visitor authentication settings" />
+                    <input
+                        label="Include claims in JWT token"
+                        hint="Add user & custom claims from Auth0 backend to the JWT token to enrich the user's experience while navigating the site"
```

Comment:
This description should be revisited to discuss the goal, not what it does technically